tests/vocabularies_tests.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import os
+import json
+from superdesk.tests import TestCase
+from superdesk import get_resource_service
+from superdesk.vocabularies.command import VocabulariesPopulateCommand
+
+
+class VocabulariesPopulateTest(TestCase):
+
+    def setUp(self):
+        super().setUp()
+        self.filename = os.path.join(os.path.abspath(os.path.dirname(__file__)), "vocabularies.json")
```

Comment:
Yes, we need to do something like that. Actually we have to check what values are supposed to change any keep those in the file, move static ones out.